docker-compose.prod.yml

```diff
@@ -41,40 +41,31 @@ services:
       db:
         condition: service_healthy
     volumes:
-      - ./thesis_uploads:/thesis_application_uploads
-      - ./mailing_templates:/mailing_templates
+      - ./thesis_uploads:/uploads
     expose:
       - "8080"
     environment:
       - TZ=Europe/Berlin
-      - ENVIRONMENT
-      - SPRING_DATASOURCE_URL
+      - SPRING_JPA_HIBERNATE_DDL_AUTO=update
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://db:5432/${SPRING_DATASOURCE_DATABASE}
```

Comment:
I dont really see a reason why the database host needs to be configured in the environment variables because the database host is set in the docker compose file so it makes sense to set the same name in the same config and not have different places for the same name

Reply:
You are correct with the "same place" 👍 I just feel like the port should be configurable in one place and not in `server` and `db`

Reply:
mmh but the port is completely internal so whatever you would put in the environment variable would be used as port and would not make any difference as database cant be accessed from the outside

Reply:
I feel quite strong about the way I did it here